apps/dashboard/src/components/workflow-editor/steps/delay/delay-control-values.tsx

```diff
@@ -16,5 +18,14 @@ export const DelayControlValues = () => {
 
   const { [amountKey]: amount, [typeKey]: type, [unitKey]: unit } = uiSchema.properties ?? {};
 
-  return amount && type && unit && getComponentByType({ component: amount.component });
+  return (
+    amount &&
+    type &&
+    unit && (
+      <>
+        <SidebarContent>{getComponentByType({ component: amount.component })}</SidebarContent>
```

Comment:
This feels like a small code smell. Let me explain. I'd expect the SidebarContent and the Sidebar wrapper to be in the same file and the delay/digest control values form independent of it's hosting div.

Reply:
This component represents the "layout" for the delay form. We can't have a general `SidebarContent` and `Separator` to be defined in the parent (step sidebar) because each step has a different "layout" in the sidebar.

For example, the "digest" step has two "sections": `Aggregated by` and `Digest window` which have inner paddings and separate each "section" with a separator.
The delay step has only one "section" (as of now).